SMLHelper/Patchers/CraftDataPatcher_Subnautica.cs

```diff
@@ -58,148 +59,73 @@ private static void PatchForSubnautica(HarmonyInstance harmony)
             harmony.Patch(AccessTools.Method(typeof(CraftData), nameof(CraftData.IsBuildableTech)),
                prefix: new HarmonyMethod(AccessTools.Method(typeof(CraftDataPatcher), nameof(GetBuildablePrefix))));
 
+            harmony.Patch(AccessTools.Method(typeof(CraftData), nameof(CraftData.GetUseEatSound)),
+               prefix: new HarmonyMethod(AccessTools.Method(typeof(CraftDataPatcher), nameof(GetUseEatSoundPrefix))));
+
         }
 
         private static void GetHarvestOutputPrefix(TechType techType)
         {
-            if (CustomHarvestOutputList.TryGetValue(techType, out TechType smlho))
-            {
-                if(CraftData.harvestOutputList.TryGetValue(techType, out TechType tt))
-                {
-                    if(smlho != tt)
-                        CraftData.harvestOutputList[techType] = smlho;
-                }
-                else
-                {
-                    PatchUtils.PatchDictionary(CraftData.harvestOutputList, CustomHarvestOutputList);
-                }
-            }
+            DictionaryPrefix(techType, CustomHarvestOutputList, CraftData.harvestOutputList);
```

Comment:
Looks good :) Also, I think some simplifications can be done with `NeedsPatchingCheckPrefix` and `PatchCustomTechData`, at least with those ~30 copypasted lines.